frontend/packages/console-app/src/components/quick-starts/data/mocks/json/sample-application-quickstart.ts

```diff
@@ -1,4 +1,6 @@
-export const sampleApplicationQuickStart = {
```

Comment:
Why do we still need typescript files for quick starts if everything is converted to yamls?

Reply:
retaining most as test data for QSCatalog and QS related components. Removed a couple

Reply:
For tests I think keeping just one would be fine.

Reply:
I've now kept only 3 quickStarts as I am checking them in the QuickStart Catalog for 3 concurrent quickStarts. removed rest.